### ISSUE TITLE
Is there a way to monitor the estimated LR over time, if it has any meaning?

### ISSUE THREAD

**Issue by ethansmith2000** [konstmish/prodigy#12]
Thank you for your incredible work first of all! I am curious if there is any merit in using prodigy to estimate what may be decent LRs for a certain training. And if those can then be used to achieve similar performance with AdamW

**Comment by konstmish** [comment]
Hi @ethansmith2000, I'm happy our optimizer was useful! You can of course just take the learning rate that Prodigy estimates. You can extract it after running Prodigy with this code: `lr = optimizer.param_groups[0]['d']` and then passing `lr` as an argument to AdamW. While we haven't done anything like that before, we have observed that Prodigy often estimates a very similar learning rate to the one of AdamW.

Let me know if you have any other questions or feedback.
